libs/shared/v2/src/lib/policies/workgroup.policy.ts

```diff
@@ -9,6 +9,6 @@ export class WorkgroupPolicy extends Policy<Workgroup> {
 
   canCreate(): boolean {
     // Only admins can create workgroups.
-    return false;
+    return this.canDoEverything() || false;
```

Comment:
I would fully get rid of the `canDoEverything` method, e.g. remove it from the abstract `Policy` class. There are some spots where it is still in use, and might probably need to be removed. Here, it can just be replaced with `this.user.isAdmin()`.